src/com/goide/sdk/GoSdkService.java

```diff
@@ -74,7 +75,7 @@ public static String getGoExecutablePath(@Nullable String sdkHomePath) {
         // gcloud and standalone installations
         return sdkHomePath.endsWith(GoConstants.GCLOUD_APP_ENGINE_DIRECTORY_PATH)
                ? FileUtil.join(StringUtil.trimEnd(sdkHomePath, GoConstants.GCLOUD_APP_ENGINE_DIRECTORY_PATH), "bin", executable)
-               : FileUtil.join(sdkHomePath, executable);
+               : FileUtil.join(sdkHomePath, SystemInfo.isWindows ? StringUtil.replace(executable, ".exe", ".bat") : executable);
```

Comment:
> OK, this is amazing, GAE for Windows uses goapp.batbut gcloud for Windows uses goapp.cmd.

Haha :) That's really funny. 

> Should I create a new pull request which includes gae and gcloud for Windows correction?

Just push correction in your fork-master, the PR should be updated automatically (hope so).

And one note. There is a line 
```
String executable = GoEnvironmentUtil.getBinaryFileNameForPath(GoConstants.GAE_EXECUTABLE_NAME);
```

And it looks strange to append executable with "exe" (getBinaryNameForPath does it) and just after that replace "exe" with something else. Please inline executable value inside each FileUtil.join invocation. Maybe it makes sense to create util methods `getBinaryOrCmdFileName` and `getBinaryOrBatFileName`